2022-08-august/README.md

```diff
@@ -77,12 +77,20 @@ Feel free to reach out to anyone here on Slack, and post updates about your own
 
 Yes, I'm talking about Assemble. Or rather, Ian (14) and Sam (17) are:
 
-### TODO: Insert some catchy title for the hackathons grant>
+### 💸 Grants!!! For your hackathon!!! 💸
+![You get $500, Oprah meme](https://cloud-ncs2ypbyu-hack-club-bot.vercel.app/06qcy84.jpeg)
 
 
-### Fun things and trends
+At Assemble, <img src="https://ca.slack-edge.com/T0266FRGM-U0266FRGP-7f1bd1a815ac-512" width="16"> [Zach](https://zachlatta.com/) announced that [Hack Club and FIRST are offering a **✨ $500 grant ✨**](https://hackclub.com/hackathons/) to in-person high-school hackathons hosted before the end of 2022!!! All you need for the grant is a high-school team, a venue, and a custom website, and you get free tools and support too. Learn more at [hackclub.com/hackathons](https://hackclub.com/hackathons) and [APPLY HERE!](https://airtable.com/shrpYdJpAOphEWXan)
 
-TODO
+
+
+### Things to do while procrastinating on math homework:
+- Add your own Orpheus drawing (there are HUNDREDS!!): [github.com/hackclub/dinosaurs](https://github.com/hackclub/dinosaurs)
+- Grab some popcorn 🍿 and join [#movie-nights](https://hackclub.slack.com/archives/C025QEVNRPW) or share your barebones web creations 🕸 in [#freewrite](https://hackclub.slack.com/archives/C03TE1RLR7V)
```

Comment:
can this be two separate points? wasn't clear to me what #freewrite is about, so a separate point will have more room to explain

Reply:
yes! what about 
`- Code something small and special in html, css, and/or js (no frameworks) and share it with the community through [freewrite](https://hackclub.slack.com/archives/C03TE1RLR7V/p1660684775476709)`

Reply:
looks good, go ahead